test/llvm/abi/x86-64/export-vs-c.good

```diff
@@ -155,7 +155,7 @@ define weak dso_local void @int8_arg_chapel(i8 signext)
 define weak dso_local void @struct_aligned16test_arg_c_____(%struct.aligned16test* byval(%struct.aligned16test) align 16)
 define weak dso_local void @struct_aligned16test_arg_chapel(%struct.aligned16test* byval(%struct.aligned16test) align 16)
 define weak dso_local void @struct_aligned16test_return_c_____(%struct.aligned16test* noalias sret(%struct.aligned16test) align 16)
-define weak dso_local void @struct_aligned16test_return_chapel(%struct.aligned16test* noalias sret align 16)
```

Comment:
Should there be similar updates to test/llvm/abi/aarch64 ?

Reply:
I'm not sure - I haven't tested on an aarch64 system. I'd expect the interruption to be relatively minimal from this test if it needs similar updates though.

Reply:
I think it would be better to test and get the expected output in before merging, but maybe I'm not understanding something and I'll leave it to your discretion